### PR TITLE
130 submit page

### PR DESCRIPTION
**Pull request by MateuszPiotrowski**

### CODE REVIEW

File: app/elements/submit-page.html [hunk 1]
```diff
@@ -67,7 +68,22 @@
     }
 
     .section__description-startup {
-        margin-top: 3.7rem;
+        margin-top: 0;
+        font-size: 1.35rem;
+        font-weight: 400;
+        line-height: 1.85;
```
Comment (krzysztofp) [inline app/elements/submit-page.html]: 1.85 px? em? rem?

File: app/elements/submit-page.html [hunk 2]
```diff
@@ -67,7 +68,22 @@
...
+
+    .section__description-startup br {
+        line-height: 3.3;
+    }
+
+    .section__description--bold {
+        display: block;
+        font-size: 1.35rem;
+        font-weight: 700;
+        line-height: 1.8;
```
Comment (krzysztofp) [inline app/elements/submit-page.html]: 1.8 px? em? rem?

File: app/elements/submit-page.html [hunk 3]
```diff
@@ -77,6 +93,7 @@
 
     .section__submit-link {
       background-color: #0099FF;
+      color: white;
       /* Overwrite MDL default style */
```
Comment (krzysztofp) [inline app/elements/submit-page.html]: Is this comment still necessary?

File: app/elements/submit-page.html [hunk 4]
```diff
@@ -797,64 +817,47 @@ <h4 class="role__name"><img src="../images/icons/share.svg" alt="Share" class="r
...
-                        <a href="http://unicefstories.org/" class="faq__link">Submit</a> your expression of interest by completing the form.<br>
-                        Preview the questions to prepare your submission, <a href="http://unicefstories.org/" class="faq__link">preview questions</a>
-                    </li>
+                    [[startups.submit_startup_step_content.3]]
                 </ul>
             </li>
         </ol>
         <div class="faq__submit">
-            <a href="https://docs.google.com/forms/d/e/1FAIpQLSdL7bKFc-zSzm4R4wK5eKsKRzW_HqQsdPYyA62sCXR84BrF9A/viewform?usp=send_form" class="faq__submit-link mdl-button mdl-js-button mdl-js-ripple-effect" target="_blank">Submission Form</a>
+            <a href="https://form.jotform.com/62704508221145" class="faq__submit-link mdl-button mdl-js-button mdl-js-ripple-effect" target="_blank">[[startups.submit_startup_submit_button]]</a>
```
Comment (krzysztofp) [inline app/elements/submit-page.html]: shouldn't the <a href> be dynamically fetched from Firebase?

File: app/elements/submit-page.html [hunk 5]
```diff
@@ -797,64 +817,47 @@ <h4 class="role__name"><img src="../images/icons/share.svg" alt="Share" class="r
...
         <hr class="section__break--thin">
         <div class="summary">
             <img src="../images/icons/country_offices-alt.svg" alt="" class="summary__icon">
             <div class="summary__content">
-                <h3 class="summary__title">Got questions?</h3>
-                <p class="summary__description">Responses will be posted regularly.</p>
-                <em class="summary__description summary__emphassis">Please note that the period to submit questions for the REOI closes on DD MONTH, YYYY.</em>
-                <p class="summary__description">UNICEF will not answer any questions submitted after this date.</p>
-                <a href="http://unicefstories.org/" class="summary__link">Ask now</a>
+                <h3 class="summary__title">[[startups.submit_startup_conclusion_title]]</h3>
```
Comment (krzysztofp) [inline app/elements/submit-page.html]: Have you considered using app/elements/echo-html.html component? You can embed HTML with it.

File: app/elements/submit-page.html (hunk 1 above)
Comment (MateuszPiotrowski) [inline app/elements/submit-page.html]: Just 1.85, without unit.

File: app/elements/submit-page.html (hunk 2 above)
Comment (MateuszPiotrowski) [inline app/elements/submit-page.html]: As above, 1.8 without unit.

**Pull request merged**